controllers/osbuild_controller.go

```diff
@@ -47,16 +81,220 @@ type OSBuildReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.11.0/pkg/reconcile
 func (r *OSBuildReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	logger := log.FromContext(ctx).WithValues("osbuild", req.Name)
+
+	osBuild, err := r.OSBuildRepository.Read(ctx, req.Name, req.Namespace)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return ctrl.Result{}, nil
+		}
+		return ctrl.Result{Requeue: true, RequeueAfter: RequeueForTwoMinutes}, err
```

Comment:
Why wait for two minutes in case of errors?

Reply:
for retring

Reply:
Sure. You should retry, but why wait 2 minutes?

Reply:
Change it to 10 seconds